torch/csrc/stable/ops.h

```diff
@@ -57,21 +57,41 @@ inline Tensor narrow(Tensor& self, int64_t dim, int64_t start, int64_t length) {
 inline Tensor new_empty(
     const Tensor& self,
     std::vector<int64_t> size,
-    std::optional<c10::ScalarType> dtype = std::nullopt) {
-  int32_t device_type;
-  TORCH_ERROR_CODE_CHECK(aoti_torch_get_device_type(self.get(), &device_type));
-
-  int32_t device_index;
-  TORCH_ERROR_CODE_CHECK(
-      aoti_torch_get_device_index(self.get(), &device_index));
-
+    std::optional<c10::ScalarType> dtype = std::nullopt,
+    std::optional<int32_t> device_type = std::nullopt,
```

Comment:
Hmm, ideally we would want this to be `device` to match native_functions but we don't have a `Device` type in stable/headeronly yet